js_modules/dagster-ui/packages/ui-core/src/assets/catalog/AssetSelectionSummaryTile.tsx

```diff
@@ -107,7 +112,30 @@ export const AssetSelectionSummaryTile = React.memo(
   },
 );
 
+const THIRTY_SECONDS = 30000;
+
+// todo dish: Move this out of asset-related code, make tile component generic.
```

Comment:
are you planning to resolve this TODO as part of this pr or a separate one?

Reply:
I'll take care of it in a followup.